nipype/interfaces/ants/segmentation.py

```diff
@@ -1153,3 +1154,160 @@ def _list_outputs(self):
+
+
+class KellyKapowskiOutputSpec(TraitedSpec):
+    cortical_thickness = File(desc="A thickness map defined in the segmented gray matter.")
+    warped_white_matter = File(desc="A warped white matter image.")
+
+
+class KellyKapowski(ANTSCommand):
+    """ Nipype Interface to ANTs' KellyKapowski, also known as DiReCT.
+
+    DiReCT is a registration based estimate of cortical thickness. It was published
+    in S. R. Das, B. B. Avants, M. Grossman, and J. C. Gee, Registration based
+    cortical thickness measurement, Neuroimage 2009, 45:867--879.
+
+    Examples
+    --------
+    >>> from nipype.interfaces.ants.segmentation import KellyKapowski
+    >>> kk = KellyKapowski()
+    >>> kk.inputs.dimension = 3
+    >>> kk.inputs.segmentation_image = "segmentation0.nii.gz"
+    >>> kk.inputs.convergence = "[45,0.0,10]"
+    >>> kk.inputs.gradient_step = 0.025
+    >>> kk.inputs.smoothing_variance = 1.0
+    >>> kk.inputs.smoothing_velocity_field = 1.5
+    >>> #kk.inputs.use_bspline_smoothing = False
+    >>> kk.inputs.number_integration_points = 10
+    >>> kk.inputs.thickness_prior_estimate = 10
+    >>> kk.cmdline # doctest: +ALLOW_UNICODE
+    u'KellyKapowski --convergence "[45,0.0,10]" \
+--output "[segmentation0_cortical_thickness.nii.gz,segmentation0_warped_white_matter.nii.gz]" \
+--image-dimensionality 3 --gradient-step 0.025000 --number-of-integration-points 10 \
+--segmentation-image "[segmentation0.nii.gz,2,3]" --smoothing-variance 1.000000 \
+--smoothing-velocity-field-parameter 1.500000 --thickness-prior-estimate 10.000000'
+
+    """
+    _cmd = "KellyKapowski"
+    input_spec = KellyKapowskiInputSpec
+    output_spec = KellyKapowskiOutputSpec
+
+    references_ = [{'entry': BibTeX("@book{Das2009867,"
+                                    "author={Sandhitsu R. Das and Brian B. Avants and Murray Grossman and James C. Gee},"
+                                    "title={Registration based cortical thickness measurement.},"
+                                    "journal={NeuroImage},"
+                                    "volume={45},"
+                                    "number={37},"
+                                    "pages={867--879},"
+                                    "year={2009},"
+                                    "issn={1053-8119},"
+                                    "url={http://www.sciencedirect.com/science/article/pii/S1053811908012780},"
+                                    "doi={http://dx.doi.org/10.1016/j.neuroimage.2008.12.016}"
+                                    "}"),
+                    'description': 'The details on the implementation of DiReCT.',
+                    'tags': ['implementation'],
+                    }]
+
+    def _parse_inputs(self, skip=None):
+        if skip is None:
+            skip = []
+        skip += ['warped_white_matter', 'gray_matter_label', 'white_matter_label']
+        return super(KellyKapowski, self)._parse_inputs(skip=skip)
+
+    def _gen_filename(self, name):
```

Comment:
I had to do this because the `output` command argument is a 2-file argument with the values of `cortical_thickness` and `warped_white_matter`:
```
--output "[segmentation0_cortical_thickness.nii.gz,segmentation0_warped_white_matter.nii.gz]"
```
Is there a better way to do this? This is the scheme of I could find from other ANTs interfaces.